scripts/configure

```diff
@@ -116,7 +116,7 @@ fi
 if [[ -z ${TOR_PASSWORD+x} ]] || [[ -z ${TOR_HASHED_PASSWORD+x} ]]; then
   echo "Generating Tor password"
   echo
-  docker pull --quiet getumbrel/tor:v0.4.1.9
```

Comment:
Why did you remove `--quiet` here?

(not arguing it shouldn't be removed, I think it should, just wondering if it related specifically to an issue)

Reply:
Tbh we can probs nuke this line completely since docker will auto pull it if it doesn't exist.

@mayankchhabra was there a reason the output was suppressed here?

Reply:
Yes, as specified in the commit message for this change, `--quiet` option is not supported in earlier docker versions and thus the configure script fails.
I'd be in favor of removing the complete docker pull.

Reply:
Ah, makes sense!

Sorry, totally missed your commit descriptions, reading though them now!

Reply:
Yeah, lets nuke it.

Reply:
> @mayankchhabra was there a reason the output was suppressed here?

I don't think there was any reason, except making the logs cleaner. We also suppress outputs while doing a `docker-compose pull` later in the configure script:

https://github.com/getumbrel/umbrel/blob/b748b7bc331f55a23e6cc5d46b4f550d0af45dec/scripts/configure#L236

Shall we also remove that too then?

Reply:
`docker-compose` did support the `--quiet` flag on my system, so I did not remove it. Whether it makes sense to reduce the output information here (and thus keep `--quiet`) is up to you guys.